fungible_asset_wrapper/sources/fungible_asset_wrapper.move

```diff
@@ -0,0 +1,168 @@
+/// Anyone can call this module to permissionlessly wrap a v1 coin into a v2 coin, and unwrap vice versa.
+module fungible_asset_wrapper::fungible_asset_wrapper {
+    use std::option;
+    use std::signer;
+    use std::string;
+
+    use aptos_std::type_info;
+    use aptos_framework::account::{Self, SignerCapability};
+    use aptos_framework::coin::{Self, Coin};
+    use aptos_framework::fungible_asset::{Self, FungibleAsset, MintRef, BurnRef};
+    use aptos_framework::object;
+    use aptos_framework::primary_fungible_store;
+    use aptos_framework::resource_account;
+
+    #[test_only]
+    use aptos_framework::coin::{MintCapability, BurnCapability};
+    #[test_only]
+    use aptos_framework::fungible_asset::Metadata;
+
+    const RESOURCE_ACCOUNT_ADDRESS: address = @fungible_asset_wrapper;
+    const DEPLOYER_ADDRESS: address = @deployer;
+    const RESOURCE_ACCOUNT_SEED: vector<u8> = b"fungible_asset_wrapper";
+
+    struct SignerCapBox has key {
+        signer_cap: SignerCapability
+    }
+
+    struct FungibleAssetAdmin has key {
```

Comment:
This should be {option<MintRef>, ....}

Reply:
@lightmark sorry i am not following - can you elaborate?

Reply:
I was wrong. forget about it.